src/command_modules/azure-cli-profile/azure/cli/command_modules/profile/_params.py

```diff
@@ -24,4 +24,4 @@ def get_subscription_id_list(prefix, **kwargs):#pylint: disable=unused-argument
 register_cli_argument('logout', 'username', help='account user, if missing, logout the current active account')
 
 register_cli_argument('account', 'subscription_name_or_id', options_list=('--name', '-n'), help='Name or ID of subscription.', completer=get_subscription_id_list)
```

Comment:
`One account == One subscription`, but I have started to felt the `subscription` looks better. Let us go ahead